### PR TITLE
Fixed almost all of e2e tests

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: neptune/new/internal/backends/hosted_neptune_backend.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def get_metadata_container(
                 .result
             )
 
-            if experiment.type != container_type:
+            if experiment.type != container_type.value:
```
Comment (pkasprzyk) [inline neptune/new/internal/backends/hosted_neptune_backend.py:284]: You could actually change `ContainerType` class from `class ContainerType(enum.Enum)` to `class ContainerType(str, enum.Enum)` and solve such problems once and for all. Makes sense?

File: neptune/new/handler.py [hunk 2]
```diff
@@ -454,7 +454,9 @@ def download_last(self, destination: str = None) -> None:
         .. _download_last docs page:
            https://docs.neptune.ai/api-reference/field-types#.download_last
         """
-        return self._pass_call_to_attr(function_name="remove", destination=destination)
+        return self._pass_call_to_attr(
+            function_name="download_last", destination=destination
```
Comment (pkasprzyk) [inline neptune/new/handler.py:458]: :open_mouth:

File: e2e_tests/conftest.py [hunk 3]
```diff
@@ -44,6 +45,8 @@ def environment():
         api_token=admin_token,
     )
 
+    time.sleep(10)
```
Comment (pkasprzyk) [inline e2e_tests/conftest.py:48]: is it really necessary? :(

10s wait in almost(?) every test is a lot...

Comment (Raalsky) [reply e2e_tests/conftest.py:48]: huh? Not in every. Only once or twice as it's inside of session-wide pytest fixture

Comment (pkasprzyk) [reply e2e_tests/conftest.py:48]: Right, I've missed that `scope="session"`. Sorry.

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**